### PR TITLE
Non-substantive changes per recommendations

### PR DESCRIPTION
**Pull request by dathbe**
Non-substantive changes as recommended here https://github.com/KristjanESPERANTO/MagicMirror-3rd-Party-Modules/blob/main/result.md#MMM-WeatherAlerts-by-gjonesme

They also recommend:

* Replacing jshint with eslint
* Replacing eslintrc with flat config
* Replacing grunt with "something better"

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,17 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/gjonesme/MMM-WeatherAlerts.git"
+  },
+  "keywords": [
+    "MagicMirror",
+    "MagicMirror²",
+    "Weather",
+    "Alerts",
+    "OpenWeatherMap",
```
Comment (KristjanESPERANTO) [inline package.json:15]: Trailing comma is not allowed in JSON files. With that the file can not be parsed. Please remove the comma at the end of line 15 :slightly_smiling_face:

**Review by KristjanESPERANTO — CHANGES_REQUESTED** [review]
Except the comma it looks good!  Thanks for your work! :rocket:

File: README.md [hunk 2]
```diff
@@ -24,6 +24,14 @@ git clone https://github.com/gjonesme/MMM-WeatherAlerts
 cd MMM-WeatherAlerts && npm install
 ```
 
+# Update
+
+```
```
Comment (KristjanESPERANTO) [inline README.md:29]: I recommend adding `bash` after the backticks. Checkout [MD040](https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md040.md).

Sorry if I'm being too picky! :smile:

**Review by KristjanESPERANTO — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (dathbe) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -16,19 +16,27 @@ OpenWeatherMap currently advertises version 3.0 of their onecall API which requi
...
+git pull
+npm install
+```
+
 # Usage
 
 To use this module, add it to the modules array in the config/config.js file:
 
-```
+```bash
```
Comment (KristjanESPERANTO) [inline README.md:39]: Please replace "bash" by "js" here 🙂

**Review by KristjanESPERANTO — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (dathbe) [reply README.md]: I'll get it right eventually.

**Comment by dathbe** [comment]
I think I've resolved everything here.  If not, let me know.

**Review by KristjanESPERANTO — APPROVED** [review]

**Pull request merged**